shared/chat/conversation/messages/coinflip/container.js

```diff
@@ -36,6 +36,9 @@ const mapStateToProps = (state, {flipGameID}: OwnProps) => {
 export default namedConnect<OwnProps, _, _, _, _>(
   mapStateToProps,
   (d, o) => ({}),
-  (s, d, o) => ({...s}),
+  (s, d, o) => ({
+    ...s,
+    howThisWorksURL: 'https://keybase.io/coin-flip',
+  }),
```

Comment:
Is there any reason to do it this way (instead of hard-coding it on the `Text` element?